server.js

```diff
@@ -1,9 +1,19 @@
-const express = require('express');
+const express = require("express");
+const mongoose = require("mongoose");
 
 const app = express();
 
-const port = 3000;
+const PORT = 3000;
+const DB_URL =
+  "mongodb+srv://ravi123:Abc123@cluster0.fbt0o.mongodb.net/internX";
```

Comment:
Let's move username/password to a `.env` file.

Reply:
I changed it. it's moved to the.env file